src/cloud/lib/i18n/enUS.ts

```diff
@@ -405,10 +405,10 @@ const enTranslation: TranslationSource = {
   [lngKeys.GeneralContinueVerb]: 'Continue',
   [lngKeys.GeneralShared]: 'Shared',
   [lngKeys.GeneralRenameVerb]: 'Rename',
-  [lngKeys.GeneralEditVerb]: 'Edit',
+  [lngKeys.GeneralEditVerb]: 'Settings',
```

Comment:
Personal preference but I disagree with this change, Edit is clear of intent and function for users.
Settings is not, especially since we have other settings in place ( user and team ) within sidebar as well. 
Translation in some languages also falters with this ( French for example would never use this word in this context for example )

Reply:
Let's wait for user feedback